src/HostRouter.ts

```diff
@@ -29,6 +29,23 @@ export class HostRouter {
 
     return clientUrl;
   }
+
+  /**
+   * Gets the client id for a provided route.
+   *
+   * @param route The route to lookup, such as '/foo/bar/baz'
+   */
+  public getClientId(route: string): string | null {
```

Comment:
I would like to merge this into getClientUrl before pushing.

Reply:
Yeah, I think something like getClientInfo makes more sense once we start doing both.

Reply:
I changed this around.  Although it was a tad confusing, since ClientInfo is the type used by the original maps.

I considered returning a ClientInfo, but the assignedRoute and url were both different from the original.  So i thought that would be confusing.  Instead I went with a non-defined type with just the id/url.  Its slightly ugly, but I think its more clear from a consumer perspective.